firewall/filesystem_storage_firewall.go

```diff
@@ -0,0 +1,41 @@
+package firewall
+
+import (
+	"errors"
+)
+
+type FilesystemAcraFirewall struct {
```

Comment:
does this firewall use any filesystem specific logic? if not then will be good to remove `filesystem` prefix and name just `Firewall` (we haven't any rules to add prefixes like `acra` everywhere :)

Reply:
Agree. Done.